### PR TITLE
Add spaces route

### PR DESCRIPTION
**Pull request by JonathanAndrews**
In this branch, we have implemented a page in which the user can add a new space to the database. Once they have filled in the form, this will be added to our database and then be redirected to the dashboard with the new listing showing up at the bottom.

### CODE REVIEW

**Comment by alakijaayo** [comment]
🇦🇷

File: index.js [hunk 1]
```diff
@@ -39,6 +39,31 @@ app.get('/dashboard', (req, res) => {
     }
   });
 });
+app.post('/login', (req, res) => {
+  req.session.email = req.body['loginemail']
+  req.session.password = req.body['loginpassword']
```
Comment (Kharouk) [inline index.js]: nice

File: index.js [hunk 2]
```diff
@@ -39,6 +39,31 @@ app.get('/dashboard', (req, res) => {
...
+  res.redirect('/dashboard');
+});
+
+  // this takes you to New Space form
+app.get('/listings/new', (req, res) => {
```
Comment (Kharouk) [inline index.js]: wow

File: index.js [hunk 3]
```diff
@@ -62,15 +87,12 @@ app.get('/listing/:id', (req, res) => {
...
-});
 
 // app.post('/login', (req, res) => {});
 
 // Connecting to our localhost
 app.listen(port, () => {
   console.log(`App listening on port ${port}`);
 });
+
+// jjjdjdjdjjdjdjdjdjdjdjdjdjdjdj
```
Comment (Kharouk) [inline index.js]: bad

File: views/insert_space.ejs [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  <h1>Please Input Info:</h1>
+</header>
+
+<body>
+  <!-- This is the form for adding speces to our space database and dashboard list -->
+  <form action="/listings/new" method="post">
+    <input type="text" name="name">
+    <input type="text" name="description">
+    <input type="text" name="price">
+    <button type="sumbit" name="Submit">Submit</button>
```
Comment (Kharouk) [inline views/insert_space.ejs]: sumbit

**Pull request merged**